guides/telephony/twilio-daily-webrtc.mdx

```diff
@@ -1,197 +1,265 @@
 ---
 title: "Dial-in: WebRTC (Twilio + Daily)"
-description: "Call your Pipecat bot using Daily WebRTC and a Twilio phone number"
+description: "Connect phone calls to your Pipecat bot using Twilio and Daily's SIP integration"
 ---
 
 ## Things you'll need
 
-- An active [Twilio](https://www.twilio.com) developer key.
-- One or more Twilio provisioned phone numbers (covered below).
-- The Twilio Python client library (`pip install twilio`).
+- An active [Twilio](https://www.twilio.com) developer key
+- One or more Twilio provisioned phone numbers (covered below)
+- The Twilio Python client library (`pip install twilio`)
 
 <Card
   title="Prefer to look at code? See the example project!"
   icon="code"
-  href="https://github.com/pipecat-ai/pipecat/tree/main/examples/phone-chatbot"
+  href="https://github.com/pipecat-ai/pipecat/tree/main/examples/phone-chatbot-daily-twilio-sip"
 >
-  We have a complete dialin-ready project using Daily as both a transport and
-  Twilio as a phone provider in the Pipecat repo. This guide references the
-  project and steps through the important parts that make dial-in work.
+  We have a complete example project using Daily as a transport with Twilio as a
+  phone provider. This guide walks through the important components and best
+  practices when integrating voice calls.
 </Card>
 
+## How It Works
+
+Here's the sequence of events when someone calls your Twilio number:
+
+1. Twilio receives an incoming call to your phone number
+2. Twilio calls your webhook server (`/call` endpoint)
+3. Your server creates a Daily room with SIP capabilities
+4. Your server starts the bot process with the room details
+5. Your server responds to Twilio with TwiML that puts the caller on hold with music
+6. The bot joins the Daily room and signals readiness via the `on_dialin_ready` event
+7. The bot forwards the call to Daily's SIP endpoint
```

Comment:
The bot updates Twilio's twiml with Daily's sip address. Twilio takes the call of hold and forward the call.